### PR TITLE
webapp/*.jsp: change to 2-space indentation #7352

### PR DESCRIPTION
**Pull request by sujeet14108**
Fixes (partially) #7352
Have applied to jsp files in `src/main/webapp/*` .

### CODE REVIEW

File: src/main/webapp/about.jsp [hunk 1]
```diff
@@ -1,334 +1,2132 @@
...
+      <h3>Past Committers:</h3>
+      <ol>
+        <li>John Kevin Tjahjadi (Jan 2016 - May 2016)</li>
+        <li>Kenneth Ho Chee Chong (Jan 2016 - May 2016)</li>
+        <li>Su Sumei (Jan 2016 - May 2016)</li>
+        <li>Wong Yu Xuan (Jan 2016 - May 2016)</li>
+      </ol>
+    </section>
 
+<<<<<<< HEAD
```
Comment (whipermr5) [inline src/main/webapp/about.jsp]: Merge conflict detected

File: src/main/webapp/about.jsp [hunk 2]
```diff
@@ -1,334 +1,2132 @@
...
+    <section class="row">
+      <h3>Contributors:</h3>
+      <table class="ordered-table">
+        <tr class="skip"><th colspan="2">Multiple contributions</th></tr>
```
Comment (whipermr5) [inline src/main/webapp/about.jsp]: I guess the `th` should be moved to its own line.

File: src/main/webapp/about.jsp [hunk 3]
```diff
@@ -1,334 +1,2132 @@
...
+        </tr>
+        <tr>
+          <td>Zhu Liang</td>
+          <td>
+            <a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3Aparadite"
+                target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements
+            </a>
+          </td>
+        </tr>
+        <tr class="skip"><th colspan="2">One-time contributions</th></tr>
```
Comment (whipermr5) [inline src/main/webapp/about.jsp]: Same here.

File: src/main/webapp/about.jsp [hunk 4]
```diff
@@ -1,334 +1,2132 @@
...
+            </a>
+          </td>
+        </tr>
+        <tr>
+          <td>Billy Zafack</td>
+          <td>
+            <a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3ABillyZafack"
+                target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements
+            </a>
+
```
Comment (whipermr5) [inline src/main/webapp/about.jsp]: Wrong indentation level here?

Comment (whipermr5) [inline src/main/webapp/about.jsp]: Marking this.

File: src/main/webapp/about.jsp [hunk 5]
```diff
@@ -1567,4 +3365,5 @@
         <p>
             Would you like to join the TEAMMATES team as a contributor? <a href="contact.jsp">Contact Us</a>.
         </p>
+>>>>>>> master
```
Comment (whipermr5) [inline src/main/webapp/about.jsp]: Marking this.

File: src/main/webapp/about.jsp (hunk 4 above)
Comment (whipermr5) [inline src/main/webapp/about.jsp]: Marking this too.

Comment (whipermr5) [inline src/main/webapp/about.jsp]: Marking this too.

Comment (sujeet14108) [inline src/main/webapp/about.jsp]: 4 spaces as we decided in previous PR for part of the HTML tag ?

Comment (whipermr5) [inline src/main/webapp/about.jsp]: Hmm same bug; I meant to mark [line 2131](https://github.com/TEAMMATES/teammates/pull/7623/files#diff-124c11a41ef6af84b5e87c462120dac9R2131).

Comment (sujeet14108) [inline src/main/webapp/about.jsp]: There was my mistake while merging conflicts which is resolved now.
Also already checked files manually

File: src/main/webapp/enableJs.jsp [hunk 6]
```diff
@@ -1,146 +1,146 @@
 <div>
   <h1>
-    TEAMMATES works best with JavaScript enabled.<br>
-    You may enable it in your browser by following steps below.
+  TEAMMATES works best with JavaScript enabled.<br>
+  You may enable it in your browser by following steps below.
```
Comment (whipermr5) [inline src/main/webapp/enableJs.jsp]: This should be indented (the original indentation was correct).

File: src/main/webapp/enableJs.jsp [hunk 7]
```diff
@@ -1,146 +1,146 @@
...
-        <li>Locate Scripting near the bottom of the list.</li>
-        <li>Under Active Scripting, select Enable, then hit OK.</li>
-        <li>Answer yes to the following confirmation box.</li>
-        <li>Hit OK to close the Internet Options window.</li>
-        <li>Finally, refresh your browser.</li>
-      </ol>
-    </div>
+    <h3>PC</h3>
+    <ol>
+    <li>In the Tools drop-down menu, select Internet Options.</li>
```
Comment (whipermr5) [inline src/main/webapp/enableJs.jsp]: This (and the subsequent `li`s) should be indented.

File: src/main/webapp/enableJs.jsp [hunk 8]
```diff
@@ -1,146 +1,146 @@
...
-        <li>From the drop-down menu, select Preferences...</li>
-        <li>Select the Content icon/tab at the top on the window.</li>
-        <li>Then, check the Enable JavaScript checkbox.</li>
-        <li>Close the Options window to save your changes.</li>
-        <li>Finally, refresh your browser.</li>
-      </ol>
-    </div>
+    <h3>PC</h3>
+    <ol>
+    <li>In the Tools drop-down menu, select Options.</li>
```
Comment (whipermr5) [inline src/main/webapp/enableJs.jsp]: Same here.

File: src/main/webapp/enableJs.jsp [hunk 9]
```diff
@@ -1,146 +1,146 @@
...
+    <li>Next, select the Content icon/tab at the top on the window.</li>
+    <li>Then, check the Enable JavaScript checkbox under the Web Content category.</li>
+    <li>Hit OK to close the Options window and save your changes.</li>
+    <li>Finally, refresh your browser.</li>
+    </ol>
+    <h3>MAC</h3>
+    <ol>
+    <li>Select the Firefox menu item from the Apple/System bar at the top of the screen.</li>
```
Comment (whipermr5) [inline src/main/webapp/enableJs.jsp]: Same here.

File: src/main/webapp/enableJs.jsp [hunk 10]
```diff
@@ -1,146 +1,146 @@
...
-        <li>From the drop-down menu, select Preferences.</li>
-        <li>Select the Content icon/tab at the top of the window.</li>
-        <li>Then, check the Enable JavaScript checkbox.</li>
-        <li>Close the window to save your changes.</li>
-        <li>Finally, refresh your browser.</li>
-       </ol>
-    </div>
+    <h3>PC</h3>
+    <ol>
+    <li>In the Edit drop-down menu at the top of the window, select Preferences...</li>
```
Comment (whipermr5) [inline src/main/webapp/enableJs.jsp]: Same here.

File: src/main/webapp/enableJs.jsp [hunk 11]
```diff
@@ -1,146 +1,146 @@
...
+    <li>Select the Security icon/tab at the top on the window.</li>
+    <li>Then, check the Enable JavaScript checkbox.</li>
+    <li>Close the window to save your changes.</li>
+    <li>Finally, refresh your browser.</li>
+    </ol>
+    <h3>MAC</h3>
+    <ol>
+    <li>Select the Safari menu item from the Apple/System bar at the top of the screen.</li>
```
Comment (whipermr5) [inline src/main/webapp/enableJs.jsp]: Same here.

File: src/main/webapp/enableJs.jsp [hunk 12]
```diff
@@ -1,146 +1,146 @@
...
+    <li>From the drop-down menu, select Preferences.</li>
+    <li>Select the Content icon/tab at the top of the window.</li>
+    <li>Then, check the Enable JavaScript checkbox.</li>
+    <li>Close the window to save your changes.</li>
+    <li>Finally, refresh your browser.</li>
+     </ol>
```
Comment (whipermr5) [inline src/main/webapp/enableJs.jsp]: Should be one space to the left.

File: src/main/webapp/enableJs.jsp [hunk 13]
```diff
@@ -1,146 +1,146 @@
...
-        <li>From the drop-down menu, select Preferences.</li>
-        <li>Select the Content icon/tab at the top of the Preferences window.</li>
-        <li>Then, check the Enable JavaScript checkbox.</li>
-        <li>Click OK to save your changes and close the Preferences window.</li>
-        <li>Finally, refresh your browser.</li>
-      </ol>
-    </div>
+    <h3>PC</h3>
+    <ol>
+    <li>In the Tools drop-down menu at the top of the window, select Preferences...</li>
```
Comment (whipermr5) [inline src/main/webapp/enableJs.jsp]: This (and the subsequent `li`s) should be indented.

File: src/main/webapp/enableJs.jsp [hunk 14]
```diff
@@ -1,146 +1,146 @@
...
+    <li>Select the Advanced tab at the top on the Preferences window.</li>
+    <li>Find the Content item in the list on the left-side of the window and select it.</li>
+    <li>Then, check the Enable JavaScript checkbox.</li>
+    <li>Click OK to save your changes and close the Preferences window.</li>
+    <li>Finally, refresh your browser.</li>
+    </ol>
+    <h3>MAC</h3>
+    <ol>
+    <li>Select the Safari menu item from the Apple/System bar at the top of the screen.</li>
```
Comment (whipermr5) [inline src/main/webapp/enableJs.jsp]: Same here.

File: src/main/webapp/enableJs.jsp [hunk 15]
```diff
@@ -1,146 +1,146 @@
...
-        <li>Select the Under the Hood tab at the top of the window.</li>
-        <li>Under the Privacy heading, select the Content settings button.</li>
-        <li>On the left, under the features heading, select JavaScript.</li>
-        <li>Select the Allow all sites to run JavaScript radio button.</li>
-        <li>Finally, close both preference windows, and refresh the browser.</li>
-      </ol>
-    </div>
+    <h3>PC</h3>
+    <ol>
+    <li>Select Customize and control Google Chrome (wrench Icon) to the right of the address bar.</li>
```
Comment (whipermr5) [inline src/main/webapp/enableJs.jsp]: Same here.

File: src/main/webapp/enableJs.jsp [hunk 16]
```diff
@@ -1,146 +1,146 @@
...
+    <li>From the drop-down menu, select Options Select the Under the Hood tab at the top of the window.</li>
+    <li>Under the Privacy heading, select the Content settings button.</li>
+    <li>On the left, under the features heading, select JavaScript.</li>
+    <li>Select the Allow all sites to run JavaScript radio button.</li>
+    <li>Finally, close both preference windows, and refresh the browser.</li>
+    </ol>
+    <h3>MAC</h3>
+    <ol>
+    <li>Select the Chrome menu item from the Apple/System bar at the top of the screen.</li>
```
Comment (whipermr5) [inline src/main/webapp/enableJs.jsp]: Same here.

File: src/main/webapp/enableJs.jsp [hunk 17]
```diff
@@ -1,146 +1,146 @@
...
+    <li>On the left, under the features heading, select JavaScript.</li>
+    <li>Select the Allow all sites to run JavaScript radio button.</li>
+    <li>Finally, close both preference windows, and refresh the browser.</li>
+    </ol>
+  </div>
   </div>
 
   <p>
-    As taken from:
+  As taken from:
```
Comment (whipermr5) [inline src/main/webapp/enableJs.jsp]: Original indentation was correct.

File: src/main/webapp/enableJs.jsp [hunk 18]
```diff
@@ -1,146 +1,146 @@
...
   <ol>
-    <li>
-      <a href="http://activatejavascript.org/en/instructions/" target="_blank" rel="noopener noreferrer">http://activatejavascript.org/en/instructions/</a>
-    </li>
-    <li>
-      <a href="https://support.mozilla.org/en-US/kb/javascript-settings-for-interactive-web-pages" target="_blank" rel="noopener noreferrer">
-        https://support.mozilla.org/en-US/kb/javascript-settings-for-interactive-web-page
-      </a>
-    </li>
+  <li>
```
Comment (whipermr5) [inline src/main/webapp/enableJs.jsp]: Original indentation was correct.

File: src/main/webapp/enableJs.jsp [hunk 19]
```diff
@@ -5,128 +5,128 @@
...
+    </ol>
+    <h3>MAC</h3>
+    <ol>
+      <li>Select the Safari menu item from the Apple/System bar at the top of the screen.</li>
+      <li>From the drop-down menu, select Preferences.</li>
+      <li>Select the Content icon/tab at the top of the window.</li>
+      <li>Then, check the Enable JavaScript checkbox.</li>
+      <li>Close the window to save your changes.</li>
+      <li>Finally, refresh your browser.</li>
+     </ol>
```
Comment (whipermr5) [inline src/main/webapp/enableJs.jsp]: Should be one space to the left.

File: src/main/webapp/enableJs.jsp [hunk 20]
```diff
@@ -5,128 +5,128 @@
...
 
   <div>
-    <ul>
-      <li><a href="#ie">Internet Explorer</a></li>
-      <li><a href="#firefox">Mozilla Firefox</a></li>
-      <li><a href="#safari">Safari</a></li>
-      <li><a href="#opera">Opera</a></li>
-      <li><a href="#chrome">Chrome</a></li>
-    </ul>
+  <ul>
```
Comment (whipermr5) [inline src/main/webapp/enableJs.jsp]: Just noticed this... original indentation was correct.

File: src/main/webapp/enableJs.jsp [hunk 21]
```diff
@@ -5,128 +5,128 @@
...
+    <li><a href="#opera">Opera</a></li>
+    <li><a href="#chrome">Chrome</a></li>
+  </ul>
   </div>
 
   <div>
-    <div id="ie">
-      <h1>Internet Explorer</h1>
-      Please complete the following instructions to activate and enable JavaScript in Internet Explorer.
+  <div id="ie">
```
Comment (whipermr5) [inline src/main/webapp/enableJs.jsp]: Same here.

File: src/main/webapp/enableJs.jsp [hunk 22]
```diff
@@ -5,128 +5,128 @@
...
+    <ol>
+      <li>Select the Chrome menu item from the Apple/System bar at the top of the screen.</li>
+      <li>From the drop-down menu, select Preferences...</li>
+      <li>Select the Under the Hood tab at the top of the window.</li>
+      <li>Under the Privacy heading, select the Content settings button.</li>
+      <li>On the left, under the features heading, select JavaScript.</li>
+      <li>Select the Allow all sites to run JavaScript radio button.</li>
+      <li>Finally, close both preference windows, and refresh the browser.</li>
+    </ol>
+  </div>
```
Comment (whipermr5) [inline src/main/webapp/enableJs.jsp]: If the above is fixed, this should end up two spaces to the right.

Comment (sujeet14108) [inline src/main/webapp/enableJs.jsp]: Have a look :)
Does it look right now ?

File: src/main/webapp/features.jsp [hunk 23]
```diff
@@ -1,190 +1,190 @@
...
+          TEAMMATES can support a course of any size (even beyond 1,000 students), as long as you divide the students into sections of no more than 100 students.
+          There is no limit on the number of courses or sessions you could have either.
         </div>
+        <img class="row img-responsive" src="images/raised-edge.png">
+      </div>
+    </section>
+
+    <div class="row">
+      <p class="col-xs-12 col-sm-8 col-sm-offset-2">
+      For more details about these features, visit the <a href="instructorHelp.jsp" target="_blank" rel="noopener noreferrer">Instructor Help</a> page or <a href="contact.jsp">email us</a>.
```
Comment (whipermr5) [inline src/main/webapp/features.jsp]: This needs indentation.

File: src/main/webapp/request.jsp [hunk 24]
```diff
@@ -1,15 +1,15 @@
...
-                width="760px" height="880px" frameborder="0" marginheight="0" marginwidth="0">
-            Loading...
-        </iframe>
-    </div>
+  <c:set var="googleDocURL" value="https://docs.google.com/forms/d/e/1FAIpQLSfmiNsVnVANdB1-cOwkfn9l8Ts8eN-CtolLQwi93Nrug0sngw/viewform?embedded=true&formkey=dDNsQmU4QXVYTVRhMjA2dEJWYW82Umc6MQ"></c:set>
+  <h1 class="color_orange">
+    Request for an Account
+  </h1>
+  <div id="contentHolder">
+  <p> Cannot see the request form below? Click <a href="${googleDocURL}" target="_blank" rel="noopener noreferrer">here.</a></p>
```
Comment (whipermr5) [inline src/main/webapp/request.jsp]: This needs indentation.

File: src/main/webapp/studentHelp.jsp [hunk 25]
```diff
@@ -1,61 +1,61 @@
...
+            In that case, email <a href="mailto:teammates@comp.nus.edu.sg">teammates@comp.nus.edu.sg</a> so that we can reconfigure your account to use the new Gmail address.
+          </li>
+        </ol>
+      </div>
+    </li>
+    <br>
+    <li>
+      <div>
+        <b>What should I do if I'm unable to submit my responses?</b>
+       </div>
```
Comment (whipermr5) [inline src/main/webapp/studentHelp.jsp]: One space to the left.

File: src/main/webapp/studentHelp.jsp [hunk 26]
```diff
@@ -1,61 +1,61 @@
...
+       <div>
+         Email <a href="mailto:teammates@comp.nus.edu.sg">us</a> so that we can help you submit.
+         Note that we cannot do much if the submission deadline is over. It is not up to us (the TEAMMATES team) to accept overdue submissions.
+       </div>
```
Comment (whipermr5) [inline src/main/webapp/studentHelp.jsp]: If the above is fixed, this should be one space to the left.

File: src/main/webapp/request.jsp (hunk 24 above)
Comment (sujeet14108) [inline src/main/webapp/request.jsp]: Originally `<div>`and `<p>` are on same level.
Are you talking about breaking line 9 ?

File: src/main/webapp/about.jsp [hunk 27]
```diff
@@ -1,1570 +1,2132 @@
...
-                </tr>
-                <tr>
-                    <td>Anirudh Jain</td>
-                    <td><a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3Asponde25"
-                        target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements</a></td>
-                </tr>
-                <tr>
-                    <td>Anthony Schneider</td>
-                    <td><a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3ABr35Ba56"
-                        target="_blank" rel="noop
```
Comment (whipermr5) [inline src/main/webapp/about.jsp]: Caught through visual inspection of the resulting page (load http://localhost:8888/about.jsp and http://teammatesv4.appspot.com/about.jsp in two tabs and switch between the two - even a single pixel difference will become obvious due to our ability to detect moving images).

Comment (whipermr5) [inline src/main/webapp/about.jsp]: Bug again; I meant to highlight [line 924](https://github.com/TEAMMATES/teammates/pull/7623/files#diff-124c11a41ef6af84b5e87c462120dac9R924).

File: src/main/webapp/features.jsp [hunk 28]
```diff
@@ -1,190 +1,190 @@
...
-                <div class="col-xs-12 col-sm-10 col-sm-offset-1">
-                    Essay questions, MCQ questions, Multiple select questions, Numeric scale questions, ‘Distribute a fixed number of points among options’ questions, questions measuring contribution in team projects, and more choices coming soon...
-                    <br><br>
-                    You can even generate MCQ options from student names, as illustrated in the above example.
-                </div>
-                <img class="img-responsive" src="images/raised-edge.png">
-            </div>
-        </section>
+    <section class="row">
+    <div class="col-xs-12 col-sm-8 col-sm-offset-2">
```
Comment (wkurniawan07) [inline src/main/webapp/features.jsp]: This `<div>...</div>` block needs one further level of indentation.

File: src/main/webapp/features.jsp [hunk 29]
```diff
@@ -1,190 +1,190 @@
...
-                <div class="col-xs-12 col-sm-10 col-sm-offset-1">
-                    Once you have a perfect set of questions configured for a session, you can reuse those questions later, without needing to configure the same questions from scratch again.
-                    <br><br>
-                    TEAMMATES also provides some session templates to choose from.
-                </div>
-                <img class="img-responsive" src="images/raised-edge.png">
-            </div>
-        </section>
+    <section class="row">
+    <div class="col-xs-12 col-sm-8 col-sm-offset-2">
```
Comment (wkurniawan07) [inline src/main/webapp/features.jsp]: Similarly here, needs one further level of indentation.

File: src/main/webapp/about.jsp (hunk 27 above)
Comment (wkurniawan07) [inline src/main/webapp/about.jsp]: This is not properly formatted.

Comment (wkurniawan07) [inline src/main/webapp/about.jsp]: This is also not properly formatted.

Comment (whipermr5) [inline src/main/webapp/about.jsp]: GitHub bug haha - which line were you actually referring to?

Comment (wkurniawan07) [inline src/main/webapp/about.jsp]: =.=
Search for "Akshay Vijayvergia" and you'll see what I mean.

Comment (wkurniawan07) [inline src/main/webapp/about.jsp]: For this one, search for "Shubham Jain"

Comment (whipermr5) [inline src/main/webapp/about.jsp]: Looks like there's a missing `</div>` before this.

Comment (whipermr5) [inline src/main/webapp/about.jsp]: Looks like there should be a `</div>` before this.

Comment (whipermr5) [inline src/main/webapp/about.jsp]: [Last line](https://github.com/TEAMMATES/teammates/pull/7623/files#diff-124c11a41ef6af84b5e87c462120dac9R2132)

Comment (sujeet14108) [inline src/main/webapp/about.jsp]: Line number?

Comment (whipermr5) [inline src/main/webapp/about.jsp]: 2132

Comment (whipermr5) [inline src/main/webapp/about.jsp]: Yup! 47 `<div>`s and only 46 `</div>`s

Comment (whipermr5) [inline src/main/webapp/about.jsp]: Is it time to bring up JSP linting? Putting up your comment here for reference: https://github.com/TEAMMATES/teammates/issues/7499#issuecomment-307691511

Comment (sujeet14108) [inline src/main/webapp/about.jsp]: Done :)

Comment (sujeet14108) [inline src/main/webapp/about.jsp]: Done

**Comment by whipermr5** [comment]
@sujeet14108 Oops, looks like you'll need to resolve conflicts with #7391.

**Comment by sujeet14108** [comment]
@whipermr5  Please have a look :)

**Pull request merged**